tests/ert_tests/gui/test_multiple_data_assimilation.py

```diff
@@ -1,8 +1,21 @@
+import numpy as np
+
 from ert_utils import ErtTest
 
 from ert_shared.models import MultipleDataAssimilation as mda
 
 
+def test_normalized_weights():
+    weights = mda.normalizeWeights([1])
+    assert weights == [1.0]
+
+    weights = mda.normalizeWeights([1, 1])
+    assert weights == [2.0, 2.0]
+
+    weights = np.array(mda.normalizeWeights([8, 4, 2, 1]))
+    assert np.reciprocal(weights).sum() == 1.0
+
+
 class MDAWeightsTest(ErtTest):
```

Comment:
could you maybe just convert this to a regular pytest and remove the need for using ErtTest?

Reply:
Done 👍